samplecode/src/main/java/com/microsoft/azure/sample/User.java

```diff
@@ -18,7 +20,9 @@
     @PartitionKey
     private String lastName;
 
-    public User(String emailAddress, String firstName, String lastName) {
+    @JsonCreator
```

Comment:
does readme in sample folder contains code snippet of User? if yes, need to update there also to align with code.

Reply:
No, there is no README for sample code.